tests/load/README.md

```diff
@@ -219,6 +208,128 @@ Execute the `setup_k8s.sh` file from the root directory and select the **delete*
+* Execute the `setup_k8s.sh` file from the root directory and select the **create** 
+  option, in order to initiate the process of creating a cluster, setting up the env 
+  variables and building the docker image
+  ```shell
+  ./tests/load/setup_k8s.sh
+  ```
+* The cluster creation process will take some time. It is considered complete, once
+  an external IP is assigned to the `locust_master` node. Monitor the assignment via
+  a watch loop:
+  ```bash
+  kubectl get svc locust-master --watch
+  ```
+
+### Calibrate 
+
+Repeat steps 1 to 3, using a process of elimination to determine the maximum 
+USERS_PER_WORKER. The load tests are considered optimized when CPU and memory resources
+are maximally utilized. This step is meant to determine the maximum user count that a
+node can accommodate by observing CPU and memory usage while steadily increasing or
+decreasing the user count.
+
+#### 1. Start Load Test
+
+* In a browser navigate to `http://$EXTERNAL_IP:8089`
+  This url can be generated via command
+  ```bash
+  EXTERNAL_IP=$(kubectl get svc locust-master -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
+  echo http://$EXTERNAL_IP:8089
+  ```
+* Set up the load test parameters:
+    * ShapeClass: Default
+    * UserClasses: StoredNotifAutopushUser (This is considered the more resource hungry user)
+    * Number of users: USERS_PER_WORKER (Consult the [Autopush Load Test Spreadsheet][3] to determine a starting point)
+    * Ramp up: RAMP_UP (RAMP_UP = 600/USERS_PER_WORKER)
+    * Host: 'wss://autoconnect.stage.mozaws.net'
+    * Duration (Optional): 600s
+* Select "Start Swarm"
+
+#### 2. Stop Load Test
+
+Select the 'Stop' button in the top right hand corner of the Locust UI, after the
+desired test duration has elapsed. If the 'Run time' or 'Duration' is set in step 1,
+the load test will stop automatically.
+
+#### 3. Analyse Results
+
+**CPU and Memory Resource Graphs**
+
+* CPU and Memory usage should be less than 90% of the available capacity
+    * CPU and Memory Resources can be observed in 
+      [Google Cloud > Kubernetes Engine > Workloads][18]
+
+**Log Errors or Warnings**
+
+* Locust will emit errors or warnings if high CPU or memory usage occurs during the 
+  execution of a load test. The presence of these logs is a strong indication that the 
+  USERS_PER_WORKER count is too high
+    * Errors and Warnings emitted while Locust is stopping can be ignored. This can be 
+      caused by disconnecting too many websockets at once, which doesn't happen in 
+      production because of the use of Shape classes
+
+#### 4. Report Results
+
+* See [Distributed GCP Execution - Report Results](#4-report-results-1)
+* Only client-side measures, provided by Locust, are available for local execution
+
+#### 5. Update Shape and Script Values
+
+* WORKER_COUNT = MAX_USERS/USERS_PER_WORKER
+    * If MAX_USERS is unknown, calibrate to determine WORKER_COUNT 
```

Comment:
Note: Instructions for this are coming [SYNC-4196](https://mozilla-hub.atlassian.net/browse/SYNC-4196)

Reply:
Great! Thanks for that @Trinaa

Reply:
```suggestion
    * If `MAX_USERS` is unknown, calibrate to determine `WORKER_COUNT` 
```